Library/ViewModels/PledgeCreditCardViewModel.swift

```diff
@@ -96,16 +96,20 @@ public final class PledgeCreditCardViewModel: PledgeCreditCardViewModelInputs,
       .takeWhen(Signal.merge(cardConfiguredAsSelected, self.selectButtonTappedProperty.signal))
       .map { $0.id }
 
-    let cardIsSelected = cardAndSelectedCard
-      .map(==)
+    let cardIsSelected = Signal.merge(
+      creditCard.mapConst(false),
```

Comment:
Emitting `false` here to ensure that the card has a title even though we're not setting a selected card.